### PR TITLE
Trigger release v1.3.0

### LINKED ISSUE #186: 1.3.0 Release
## TODO for removing older stuff

- [x] DMPQueue https://github.com/polkadot-fellows/runtimes/pull/137#issuecomment-1893483027
  - [x] remove pallet https://github.com/polkadot-fellows/runtimes/pull/280
   ```
   impl cumulus_pallet_dmp_queue::Config
   ...
   DmpQueue: cumulus_pallet_dmp_queue = XY,
   ```
  - [x] clean storage data (add migration or `kill_storage`) https://github.com/polkadot-fellows/runtimes/pull/280
- [x] remove duplicated `check_sane_fees_values` / `diff_as_percent` when `polkadot-sdk@1.8.0`
- [x] remove hack for skipping `check-migrations` for Encointer (based on https://github.com/polkadot-fellows/runtimes/pull/159#issuecomment-1966836443)
- [x] remove `penpal-runtime-dotsama` and instead fix like [this](https://github.com/paritytech/polkadot-sdk/pull/3564) when `polakdot-sdk@1.9.0`
- [x] fix https://github.com/polkadot-fellows/runtimes/issues/234
- [x] check `runtimes-matrix.json` and add `uri` for `people-kusama` and `coretime-kusama` - https://github.com/polkadot-fellows/runtimes/pull/425
- [x] ~~remove `helpers` module from inegrations-tests and re-use from `polkadot-sdk` https://github.com/polkadot-fellows/runtimes/issues/363~~

### PR DESCRIPTION
**Pull request by seadanda**
Trigger the v1.3.0 release with the following additional changes:

- Bump `polkadot-runtime-parachains` version to get https://github.com/paritytech/polkadot-sdk/pull/5369 in [63cb34d](https://github.com/polkadot-fellows/runtimes/pull/438/commits/63cb34da7dd2437c1c2d47efd5b00c0820059db6) and update tests in [e718c64](https://github.com/polkadot-fellows/runtimes/pull/438/commits/e718c649ebd520a38f1990a40b5c31ed01f20c13).
- Filter calls to `interlace` on the Polkadot Coretime Chain until the relay implementation matures in [1126cf7](https://github.com/polkadot-fellows/runtimes/pull/438/commits/1126cf7c0fce7a4e1e4e3894f1a17c905116a1cc).
- Update all runtime versions to `1_003_000` in [08744df](https://github.com/polkadot-fellows/runtimes/pull/438/commits/08744dfba94fb341114bf5c8fdaec59848415854)

TODO:
- [x] Finalise changelog
- [x] Wait for https://github.com/polkadot-fellows/runtimes/pull/433
- [x] Wait for https://github.com/polkadot-fellows/runtimes/pull/364

Closes #186

### CODE REVIEW

**Review by eskimor — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Changelog for the runtimes governed by the Polkadot Fellowship.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## [Unreleased]
+## [1.3.0] 16.08.2024
```
Comment (bkchr) [inline CHANGELOG.md:7]:
```suggestion
## [1.3.0] 19.08.2024
```

**Review by bkchr — APPROVED** [review]

**Review by acatangiu — APPROVED** [review]

**Comment by ggwpez** [comment]
Would still like to get https://github.com/polkadot-fellows/runtimes/pull/364 included.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Changelog for the runtimes governed by the Polkadot Fellowship.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## [Unreleased]
+## [1.3.0] 19.08.2024
```
Comment (joepetrowski) [inline CHANGELOG.md:7]:
```suggestion
## [1.3.0] 20.08.2024
```

**Comment by kianenigma** [comment]
https://github.com/polkadot-fellows/runtimes/pull/364 was merged with `UseLegacyAuctions = false`, which means in fact there will be a small tweak to the inflation of Kusama. 

The impact is rather small, but not what the PR originally promised. 

I don't think it is big enough to escalate and re-do the release, but just FYI.

**Comment by joepetrowski** [comment]
Can you get it into 1.3.1 with https://github.com/polkadot-fellows/runtimes/pull/442 ?

**Pull request merged**